index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
+  <link rel="icon"
+    href="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/apple/237/avocado_1f951.png">
+  <title>DAILY AVOCADO</title>
+</head>
+
+<body>
+
+  <header>
+    <img src="/images/logo.svg" alt="technigo-logo">
+    <div class="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+    <div class="avocado">🥑</div>
+    <nav>
+      <a href="https://www.linkedin.com/in/johanna-%C3%A5strand/" target="_blank">RIPE NEWS</a>
+      <a href="https://www.linkedin.com/in/johanna-%C3%A5strand/" target="_blank">AVOCADO TV</a>
+      <a href="https://www.linkedin.com/in/johanna-%C3%A5strand/" target="_blank">RECIPES</a>
+    </nav>
+  </header>
+
+  <div class="latest-news">
+    <div class="latest-news-content">
+      <h2>RIGHT NOW:</h2>
+      <h3>AVOCADOS CAN FLY</h3>
+    </div>
```

Comment:
Tip! You can add `text-transform: uppercase;` in the css so you don't need to type in uppercase!